devops/sources_backup.rst

```diff
@@ -0,0 +1,122 @@
+.. _conan_backup_sources:
+
+Backup sources
```

Comment:
Probably it is worth trying a more meaningful title? Like ``Backup sources: automatically store third parties sources``?